clusters-rke2/check_etcd_perf.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+### This script will loop through all generated kubeconfig files in the "./files" directory based from wherever the script was invoked.
+### It will look for kubeconfig files matching the pattern of ".workspace-XXXXX_kube_config", where each X = a number. It will then execute the etcdctl check perf command.
+### Example invocation when executing within "clusters-rke1/linode": ../check_etcd_perf.sh ~/.ssh/<ssh key with access to all nodes> root
+### $1 - ssh key path
+### $2 - ssh user (ex: "root", "ubuntu", etc.)
+
+for workspace in $(terraform workspace list | grep "workspace")
+do
+    while read -r node
+    do
+        printf "\nGetting etcd perf results for $node\n"
+        printf "\nGetting etcd perf results for $node\n" >> "etcd_perf_${workspace}.txt"
+        ssh -n -o "StrictHostKeyChecking no" -i "${1}" -T "${2}@${node}" "sudo -s sleep 5 && docker exec etcd etcdctl check perf -w fields" >> "etcd_perf_${workspace}.txt"
```

Comment:
I don't think this is going to work because of no docker in rke2

Reply:
Correct, I accidentally included the same script used for the RKE1 modules. I'll update this for rke2.

Reply:
This should be resolved